README.md

```diff
@@ -38,7 +38,7 @@ jobs:
     steps:
       - uses: actions/checkout@v4
```

Comment:
⚠️ Verify that the new version (v1.0.0) does not introduce breaking changes compared to the previous version (sudo1). If there are changes, document them in the README to inform users.